docs/community.md

```diff
@@ -0,0 +1,14 @@
+# MicroK8s In The Wild
+
+Welcome to the spot MicroK8s In The Wild page. Here you can find, events, tutorials, blog posts and talks on MicroK8s.
```

Comment:
s/spot//

Also remove the comma after "find".

Reply:
Done